README.Rmd

```diff
@@ -151,11 +163,14 @@ nix-shell --expr "$(curl -sl https://raw.githubusercontent.com/b-rodrigues/rix/m
 
 You can then create new development environment definitions, build them, and
 start using them.
+</details>
 
 ## Getting started for new users
 
 To get started with `{rix}` and Nix, you should read the following vignette
-`vignette("a-getting-started")`. The vignettes are numbered to get you to learn
+`vignette("a-getting-started")` 
```

Comment:
we’re repeating the link here, no? the call to `vignette("a-getting-started")` will be transformed to a link by pkgdown automatically

Reply:
does this work for the README, too? Currently i don't see the link there.

Reply:
ah not on github yes, but on the pkdown’ed website it works. I guess we should add the link back then